css/device-catalog.css

```diff
@@ -0,0 +1,884 @@
+    outline: none;
+    border: none;
+    padding-top: 8px;
+    padding-bottom: 8px;
+}
+.filter-button:hover {
+    background: #f0c52e;
+}
+.filter-button:active {
+    color: rgba(0, 0, 0, 0.3);
+}
+/* FOOTER */
+
+footer {
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 1px;
+    max-width: 1440px;
+    height: auto;
+    background-color: #464646; 
+}
+
+footer div {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position: relative;
+    max-width: 1160px;
+}
+.footer-container {
+    display: block;
+    max-width: 1160px;
+    flex-direction: column;
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 65px;
+    padding-bottom: 47px;
+}
+.footer-buttons {
+    margin-bottom: 40px;
+}
+.device-yellow {
+    margin-right: auto;
+}
+.device-yellow:hover {
+    opacity: 0.6;
+}
+.device-yellow:active {
+    opacity: 0.3;
+}
+.footer-services {
+    margin-bottom: 70px;
+}
+.footer-user-buttons li:nth-child(1){
+    margin-right: 16px;
+}
+.footer-user-buttons li:nth-child(2){
+    margin-right: 45px;
+}
+.footer-user-buttons li:nth-child(3){
+    margin-right: 40px;
+}
+.footer-user-buttons a {
+    position: relative;
+    font-family: 'Opensanslight', serif;
+    font-size: 15px;
+    line-height: 30px;
+    color: rgba(255, 255, 255, 0.7)  
+}
+.footer-user-buttons a:hover {
+    /* opacity: 0.6; */
+    color: rgba(255, 255, 255, 1);
+}
+.footer-user-buttons a:active {
+    opacity: 0.3;
+}
+.user-footer {
+    padding-left: 26px;
+}
+.user-footer::before {
+    content: url(.//img/user-footer.png);
+    position: absolute;
+    left: 0px;
+}
+.user-logout {
+    opacity: 0.3;
+}
+.footer-button-compare {
+    padding-left: 20px;
+}
+.footer-button-compare::before {
+    content: url(.//img/compare-yellow.png);
+    position: absolute;
+    left: 0px;
+    
+}
+.footer-button-basket {
+    padding-left: 24px;
+}
+.footer-button-basket::before {
+    content: url(.//img/cart-yellow.png);
+    position: absolute;
+    left: 0px;
+}
+.footer-user-buttons {
+    display: flex;
+}
+.footer-services p {
+    font-family: 'Opensanslight', serif;
+    font-size: 15px;
+    line-height: 30px;
+    color: #ffffff;
+}
+.list-services {
+    display: flex;
+    flex-wrap: wrap;
+}
+.list-services li:nth-child(2) {
+    margin-left: 37px;
+    margin-right: 45px;
+}
+
+.list-services a {
+    font-size: 18px;
+    line-height: 24px;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    color: white;
+}
+.list-services a:hover {
+    opacity: 0.6;
+}
+.list-services a:active {
+    opacity: 0.3;
+}
+.footer-socials::before {
+    content: "";
+    display: block;
+    position:relative;
+    width: 80px;
+    height: 7px;
+    margin-top: auto;
+    margin-bottom: auto;
+    background-color: #ffc600;
+}
+.socials-list {
+    display: flex;
+    align-items: center;
+    position: absolute;
```

Comment:
wrong solution, Ivan. This ul definetely should be in the main flow of the document. why not to make `footer-social` container display:flex and center this ul inside of it?

Reply:
Done. Used a buffer div to justufy socials-list.